packages/devtools_extensions/README.md

```diff
@@ -164,10 +164,9 @@ some_pkg/  # formerly the repository root of your pub package
 
 2. Add the `devtools_extensions` dependency to your Flutter web app.
 
-    In `pubspec.yaml`, add the following:
-    ```yaml
-    devtools_extensions: ^0.0.14
-    ```
+   ```console
+   $ flutter pub add devtools_app_shared
```

Comment:
this needs to be devtools_extensions

Reply:
Haha oops, copy paste mistake :sweat_smile: